### PR TITLE
(doc) Minor line editing for Labelled Arguments

### PR DESCRIPTION
**Pull request by christinerose**
A few grammar / syntax / formatting changes & a few questions...

### CODE REVIEW

**Comment by christinerose** [comment]
li. 177-200 ... examples with "a" "b" "c" --- shall I change them away from those letters and into things like "m" "n" "o" to keep them from from the `'a`? @cuihtlauac

File: data/tutorials/language/0it_05_labels.md [hunk 1]
```diff
@@ -130,9 +132,9 @@ val sub : ?pos:int -> ?len:int -> string -> string = <fun>
 ```
 
 Here, we're defining a variant of the function `String.sub` from the standard library.
-* `s` is the string which we are extracting a substring from
-* `pos` is the starting position of the substring, it defaults to `0`
-* `len` is the length of the substring. If missing, it defaults to `String.length s - pos`
+* `s` is the string which we are extracting a substring from.
```
Comment (christinerose) [inline data/tutorials/language/0it_05_labels.md]:
```suggestion
* `s` is the string from which we are extracting a substring.
```
Or....  We extract a substring from the string `s`.

**Review by cuihtlauac — APPROVED** [review]

**Comment by cuihtlauac** [comment]
> li. 177-200 ... examples with "a" "b" "c" --- shall I change them away from those letters and into things like "m" "n" "o" to keep them from from the `'a`? @cuihtlauac

Pushed a commit with better strings

**Comment by christinerose** [comment]
LGTM! Shall we merge, @cuihtlauac?

**Review by cuihtlauac — APPROVED** [review]

**Pull request merged**